docker/Dockerfile-x86_64

```diff
@@ -1,9 +1,8 @@
-# See docker/glibc/
-FROM quay.io/pypa/manylinux2010_centos-6-no-vsyscall
+FROM centos:7
 LABEL maintainer="The ManyLinux project"
 
 ENV AUDITWHEEL_ARCH x86_64
-ENV AUDITWHEEL_PLAT manylinux2010_$AUDITWHEEL_ARCH
+ENV AUDITWHEEL_PLAT manylinux2014_$AUDITWHEEL_ARCH
 ENV LC_ALL en_US.UTF-8
```

Comment:
aarch64 uses C locale, x86-64 uses en_US.UTF-8

Shouldn't both use same?

Reply:
x86-64 & i686 have always used the `en_US.UTF-8` with no warnings. I changed it only on `aarch64` where multiple warnings could be seen.
Agreed they should be the same but not knowing the impact on i686 & x86_64 builds from people that will upgrade to manylinux2014, I chose to keep them as before.